### PR TITLE
Updating docs to include IKS in various places

### PR DESCRIPTION
**Pull request by artberger**
Changes include:
* Updating product name
* Adding IKS to table of solutions
* Adding the IBM Cloud Container Registry to available registries
* Adding IKS to the cloud provider page

### CODE REVIEW

**Comment by artberger** [comment]
@k8s-ci-robot new message from CNCF:
> Contribute to CNCF as an employee
> You are already authorized to contribute code to this project through your membership with CNCF - IBM.

File: content/en/docs/setup/pick-right-solution.md [hunk 1]
```diff
@@ -216,6 +216,7 @@ any                  | RKE          | multi-support    | flannel or canal
 any                  | [Gardener Cluster-Operator](https://kubernetes.io/blog/2018/05/17/gardener/) | multi-support | multi-support | [docs](https://gardener.cloud) | [Project/Community](https://github.com/gardener) and [Commercial]( https://cloudplatform.sap.com/)
 Alibaba Cloud Container Service For Kubernetes | ROS        | CentOS | flannel/Terway       | [docs](https://www.aliyun.com/product/containerservice)                    |  Commercial
 Agile Stacks       | Terraform   | CoreOS | multi-support | [docs](https://www.agilestacks.com/products/kubernetes) | Commercial
+IBM Cloud Kubernetes Service | | Ubuntu | calico | [docs](https://console.bluemix.net/docs/containers/container_index.html) | Commercial
```
Comment (artberger) [inline content/en/docs/setup/pick-right-solution.md]: The note following this table said:
> Note: The above table is ordered by version test/used in nodes, followed by support level.

However, I'm not sure what that means; there doesn't seem to be a version test column that the entries are sorted by. I just added IKS to the last row of the table. Happy to rearrange if there's a better place.

**Comment by bradtopol** [comment]
Great job!
/lgtm
/approve

**Pull request merged**